lib/peripheral.dart

```diff
@@ -28,32 +47,57 @@ class Peripheral {
           refreshGatt: refreshGatt,
           timeout: timeout);
 
+  /// Observe connection state
+  ///
+  /// Returns stream of [PeripheralConnectionState]
```

Comment:
I think it's too short to break it into two paragraphs. I would keep it in one line. And dots at the end of both lines are missing.

Reply:
When you generate docs with `dartdocs` the you can see that only the first line is visible and to see the next paragraph, you have to click `[...]` which opens a new page. I think it's too much for such a short message. :D